### DISCUSSION TITLE
Unraid ERROR VPN: custom endpoint port is not allowed: for VPN service provider privado

### DISCUSSION THREAD

**Discussion by Qballjos** [qdm12/gluetun#2346] [Q&A] [answered]
unable to setup this container with provider Privado selected, not with the template or docker compose.
It seems that it is stuck trying to use port 51820
also used custom config with privado settings and manualy changing the hostname to an ip still not picking up the wanted port 1194

**Comment by qdm12** [comment, marked as answer]
This is fixed in 36c8da7ea7152d71ba62c5f1884784793d4facde (repull the latest image or use `:v3`)
